src/main/resources/config.yml

```diff
@@ -1,3 +1,186 @@
 message-on-login-enabled: true
+default-block-material: AIR
+default-replace-block-material: AIR
+default-brush-size: 3
 litesniper-max-brush-size: 5
-litesniper-restricted-materials:
+litesniper-restricted-materials: []
```

Comment:
Add a default couple like barrier and bedrock or so to show the format being used.